frontend/public/style.scss

```diff
@@ -113,5 +113,7 @@
 @import 'components/dashboard/project-dashboard/activity-card';
 @import 'components/dashboard/project-dashboard/details-card';
 
+@import 'components/utils/favorite-button';
```

Comment:
Is there a specific reason for importing this style here? It can only be imported into the favorite component, as it is used by that component.

Reply:
All the other utils in that package followed the same standard and I don't see in any module scss imported directly, so I thought of not changing the process. I can remove if want.